src/components/transaction/history/TransactionHistoryComponent.tsx

```diff
@@ -56,9 +56,9 @@ export default class TransactionHistoryComponent extends React.Component<Props,
           onPress={() => {
             navigation.navigate(
               'TransactionDetailsTabs', {
-                params: { transactionID: transaction.id },
-                key: `${Utils.randomNumber()}`
-              }
+              params: { transactionID: transaction.id },
```

Comment:
Not indented properly!

Reply:
It's object literals passed as arguments of a method or function. 
And it's properly indented: 
+ each arguments have the same indentation level; 
+ each keys in objects literal passed as arguments have the same indentation level. 

Before the indentation level was considering only one object literal as the argument and method arguments as object literal keys, which is not correct. The re-indentation in that PR have been made automatically by code editor and linter. 

All the indentation fixlet done in that PR correct it.

Reply:
I see that, 'param' should be indented:
![image](https://user-images.githubusercontent.com/25266945/103150820-0a562f00-4778-11eb-8a03-25aa09127d94.png)

Reply:
> I see that, 'param' should be indented:
> ![image](https://user-images.githubusercontent.com/25266945/103150820-0a562f00-4778-11eb-8a03-25aa09127d94.png)

After adding a new line separator for each arguments, the automatic indentation now works.